src/index.ts

```diff
@@ -4,22 +4,29 @@ import beforeResponseMiddleware from './middlewares/beforeResponse.middleware'
 import { proxyReceiverRouter } from './app/proxy-receiver/router'
 import { testRouter } from './app/test/router'
 import { loadTestCases } from './app/test/service/testRunner'
+import { buildInfo } from "./version";
 
 const app: Express = express()
 const port = Number(process.env.PORT) || 3000
 
+app.set('view engine', 'ejs')
```

Comment:
I'm wondering if it makes sense to import the whole `ejs` dependency to replace just one variable in the file. 
We could easily replace it by ourselves, I think 🤔